doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -3914,7 +4007,7 @@ NCSDK-30263: direct-xip with revert does not work on nRF54L15
 
   **Workaround:** Manually cherry-pick the commit ``ff0e8fabe1566349dbfd1786b31b325b46be205a`` from the main branch of the ``sdk-nrf`` repository.
 
-.. rst-class:: v2-9-0-nRF54H20-1 v2-9-1 v2-9-0 v2-8-0
+.. rst-class:: v3-0-0 v2-9-0-nRF54H20-1 v2-9-1 v2-9-0 v2-8-0
```

Comment:
Should no longer apply to v3.0.0, the sha, with a solution, has already been tagged.